src/Form/test-cases/techradar_newsletter.html

```diff
@@ -0,0 +1,170 @@
+<!-- https://www.mymagazine.co.uk/AccessSubscription -->
+<form id="j_id0:theForm" name="j_id0:theForm" method="post" action="https://www.mymagazine.co.uk/AccessSubscription?refURL=http%3A%2F%2Fwww.mymagazine.co.uk%2FAccessSubscription" enctype="application/x-www-form-urlencoded">
+<input type="hidden" name="j_id0:theForm" value="j_id0:theForm">
+
+                        
+                        
+                        
+                        <div class="row small-collapse medium-uncollapse">
+                            <div class="columns large-6 small-12">
+                               
+                                    <div>
+                                        <h4>Access your subscription</h4>
+                                    </div>
+                                    <div class="row small-collapse medium-uncollapse">
+                                        <div class="small-12 large-10 columns">
+                                            <label style="font-weight: bold;">Enter Your Email<input id="j_id0:theForm:emailInput" type="text" name="j_id0:theForm:emailInput" class="trackEvent emailInput" onkeyup="searchContactEmails(j$(this));" autocomplete="off" placeholder="This is the email address used when purchasing the subscription" data-action="UA:type-email" data-category="User Account" data-manual-scoring="emailAddress" style="">
```

Comment:
I'm not sure what's happening with this case - if I run the test suite, this field is identified wrongly:

```
    console.log
      manualType:   "emailAddress" 
      inferredType: "username" 
      id:           "j_id0:theForm:emailInput" 
      name:         "j_id0:theForm:emailInput" 
      HTML:         <input id="j_id0:theForm:emailInput" type="text" name="j_id0:theForm:emailInput" class="trackEvent emailInput" onkeyup="searchContactEmails(j$(this));" autocomplete="off" placeholder="This is the email address used when purchasing the subscription" data-action="UA:type-email" data-category="User Account" data-manual-scoring="emailAddress" style="" data-ddg-inputtype="credentials.username">
```

But if I open the testcase file directly in browser, it's identified correctly:
<img width="478" alt="Screen Shot 2022-03-24 at 12 24 03 AM" src="https://user-images.githubusercontent.com/90369497/159848115-e7ef0c24-cab2-4bc9-aac2-35eec43ecf47.png">

Reply:
Setting the title ("AccessSubscription") doesn't seem to change the results for me. But I think you're right, this should be a username.